reppy/exceptions.py

```diff
@@ -36,3 +36,22 @@ def __init__(self, *args, **kwargs):
             self.status = args[1]
         ReppyException.__init__(self, *args, **kwargs)
 
+class SSLException(ReppyException):
+    def __init__(self, *args, **kwargs):
```

Comment:
For all of them -- their `__init__` simply invokes the parent, so `pass` should do the same thing less verbosely.